portfolio.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="portfolio.css"><script src="https://kit.fontawesome.com/0a62dabd9f.js" crossorigin="anonymous"></script>
+
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    
+    <header><h1>Hey there. I’m Cleophas.</br>
+        <div class="green">I’m a software developer</div>
+        </h1></header>
```

Comment:
- [ ] The content of this `<header>` tag should be a logo and the hamburger. The logo should be on the left side while the hamburger should be on the right side. Kindly move this present content to the headline section of the project and add the appropriate content to this section. Kindly check the [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md) to know what is expected of you in this section.